modules/programs/gnome-terminal.nix

```diff
@@ -234,13 +240,20 @@ let
         type = types.bool;
         description = "Turn on/off the terminal's bell.";
       };
+
+      transparencyPercent = mkOption {
+        default = null;
+        type = types.nullOr types.int;
+        description = "Background transparency percentage (0-100).";
```

Comment:
Can use a more precise type here
```suggestion
        type = types.nullOr (types.between 0 100);
        description = "Background transparency in percent.";
```

Reply:
This type may not exist in many nixpkgs version:

![image](https://user-images.githubusercontent.com/47480384/120088493-96b5a680-c0b6-11eb-869f-1095b56638c8.png)

Reply:
Try `types.ints.between`.